server/lib/subject_page_config.py

```diff
@@ -165,6 +170,8 @@ def remove_empty_charts(page_config, place_dcid, contained_place_type=None):
                                             stat_vars_existence, place_dcid)
     child_exist_keys = _exist_keys_category(sample_child_places, category,
                                             stat_vars_existence, place_dcid)
+    logging.info('_remove_empty_charts place_exist_keys: %s', place_exist_keys)
```

Comment:
nit: place_exist_keys looks to be a dict, will %s work?

```
logging.info(f'_remove_empty_charts place_exist_keys: {place_exist_keys}')
```

is another option

Reply:
yes it seems to work.